chandra_aca/maude_decom.py

```diff
@@ -197,6 +196,13 @@ def _aca_image_msid_list(pea):
         # 'image_function_pea':
         #     [f'{msid_prefix}AIMGF{i}1' for i in range(8)],  # IMAGE FUNCTION1 (PEA)
 
+        'saturated_pixel': [f'{msid_prefix}ASPXF{i}' for i in range(8)],  # DEFECTIVE PIXEL FLAG
```

Comment:
Is this related to the maude blob changes?

Reply:
Yes. When decomm'ing the frames, the MSID is not needed, but it is needed to decom from blobs. It is used in line 990 and 1004.

If the name is not good, I can change it.

Reply:
That's fine.  I was just having trouble following the various data paths without more hands-on work.  This seems fine.